### ISSUE TITLE
Making figures more suitable for screenshot usage

### ISSUE THREAD

**Issue by corneliusroemer** [GenSpectrum/cov-spectrum-website#615]
covSpectrum is so amazing, it's the most reliable and sophisticated tool to plot how variants are growing in various countries.

Unfortunately, the plots are not great for screenshot usage at the moment.

Main problem: they are too dashboardy, assume that one sees the whole screen.

But if you take a screenshot you need the figure to be dense: we're missing:
a) Title
b) Information on exact query
c) Source, date
d) country/region/qc filtering
e) good legend
f) indication of sequence numbers on parallel x axis

Not all of the above are required, addition of any single one of them would already be a great improvement.

Maybe one should set up a separate website dedicated to this kind of publication style plot, that could then be called from covSpectrum, like we call Usher etc: it would get the data for the plots from covSpectrum and output a high quality and maybe even somewhat editable figure.

Something like a Jupyter notebook with some code to generate a base figure that can be customized.

Basically what we would need to do is:
a) Get data for the above a) - f) from either the query and/or covSpectrum LAPIS
b) Use them as defaults for a matplotlib script that's running in a browser window, so one can quickly tweak it for export

This could be a great project for a student/open source contributor.

I'd be happy to give feedback and guidance.

**Comment by chaoran-chen** [comment]
Hi @corneliusroemer! I agree that print-oriented plots would be really nice to have. Ideally, I'd like to offer both versions directly on the website and I've been thinking about ways to implement it for a while. It's just that this is, unfortunately, a bigger task and without additional support, I don't think that we will be able to tackle it anytime soon.

**Comment by corneliusroemer** [comment]
Great to hear this is not just my personal crazy idea :) 

I feel like setting this up as an integration may be simpler for some outside contributors. We could try to ask for it via Twitter? Maybe someone will be interested to help? This should actually be quite easy for a large number of data scientists who are good at making graphs from the data. The hard part is the data but it's already all there, just need to export it as a nice JSON that can be imported to make these figures.

Maybe I'll make a matplotlib prototype, but R shiny app would be just as good if not better.

Maybe we could have multiple implementations!

**Comment by chaoran-chen** [comment]
We would need a server with a web API that accepts the data from the client and returns a plot. This should indeed be quite doable as a separate project.

I'll think a bit more about it.
